typings/css-element-queries.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'css-element-queries' {
```

Comment:
This TS is identical to the one in the `css-element-queries` repo, which - unfortunately - is not published on npm yet.